python-sdk/tutorials/deploy-with-prebuilt-docker-images/pytorch-prebuilt-docker-image.ipynb

```diff
@@ -0,0 +1,938 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "%%html\n",
+        "<span style=\"color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;\">An Exception was encountered at 'In [7]'.</span>"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "hide_input": true,
+        "inputHidden": true
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "%%html\n",
+        "<span style=\"color:red; font-family:Helvetica Neue, Helvetica, Arial, sans-serif; font-size:2em;\">An Exception was encountered at 'In [6]'.</span>"
+      ],
+      "outputs": [],
+      "execution_count": null,
+      "metadata": {
+        "hide_input": true,
+        "inputHidden": true,
+        "papermill": {
+          "duration": 0.133882,
+          "end_time": "2021-05-18T20:36:44.856851",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:44.722969",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Copyright (c) Microsoft Corporation. All rights reserved.\n",
+        "\n",
+        "Licensed under the MIT License."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.057557,
+          "end_time": "2021-05-18T20:36:45.043130",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:44.985573",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "# Deploy a model using Prebuilt Docker Images for Inference\n",
+        "\n",
+        "In this tutorial, you will deploy a PyTorch model using the Azure Machine Learning Python SDK. \n",
+        "The model is trained to classify chickens and turkeys by first using a pretrained ResNet18 model that has been trained on the [ImageNet](https://image-net.org/index) dataset."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        },
+        "papermill": {
+          "duration": 0.2322,
+          "end_time": "2021-05-18T20:36:45.333348",
+          "exception": false,
+          "start_time": "2021-05-18T20:36:45.101148",
+          "status": "completed"
+        },
+        "tags": []
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Prerequisites\n",
+        "\n",
+        "If you are using an Azure Machine Learning Notebook VM, you are all set. Otherwise, go through the [Configuration](../../../../configuration.ipynb) notebook to install the Azure Machine Learning Python SDK and create an Azure ML `Workspace`"
```

Comment:
notebook VMs are deprecated

Reply:
oh wow, I was taking inspiration from these notebooks and they still refer it as notebook vm
we should definitely update these notebooks as well

example:

https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/ml-frameworks/tensorflow/train-hyperparameter-tune-deploy-with-tensorflow/train-hyperparameter-tune-deploy-with-tensorflow.ipynb

https://github.com/Azure/MachineLearningNotebooks/blob/master/how-to-use-azureml/ml-frameworks/pytorch/train-hyperparameter-tune-deploy-with-pytorch/train-hyperparameter-tune-deploy-with-pytorch.ipynb